bitgouel-api/src/main/kotlin/team/msg/common/validation/NotBlankListValidator.kt

```diff
@@ -6,7 +6,7 @@ import javax.validation.ConstraintValidatorContext
 class NotBlankListValidator : ConstraintValidator<NotBlankList, List<String>> {
     override fun isValid(value: List<String>?, context: ConstraintValidatorContext): Boolean {
         if (value == null) {
-            return true
+            return false
         }
         return value.all { it.isNotBlank() }
```

Comment:
그냥 한줄로 줄일 수 있을 것 같아서 해봤습니다 괜찮으시면 적용해주세요
```suggestion
    override fun isValid(value: List<String>?, context: ConstraintValidatorContext): Boolean =
        value?.all {
            it.isNotBlank()
        } ?: false
```

Reply:
저희가 개발 시작 전 알아보기 쉬운 코드를 짜기로 했었던 걸로 기억합니다. 후자는 짧지만 알아보기 더 어려울 것 같아서 기존대로 진행하고 싶습니다.

Reply:
넵